schema/deploy/shipping/views.sql

```diff
@@ -23,6 +23,7 @@ drop view if exists shipping.scan_demographics_v1;
 
 drop view if exists shipping.uw_priority_queue_v1;
 drop view if exists shipping.__uw_priority_queue_v1;
+drop materialized view if exists shipping.__uw_encounters;
```

Comment:
Also need to include this drop statement in the revert script.

Reply:
Sorry about that. I added the drop to schema/revert/shipping/views.sql.